aws/table_aws_efs_access_point.go

```diff
@@ -0,0 +1,191 @@
+			},
+			{
+				Name:        "root_directory",
+				Description: "The directory on the Amazon EFS file system that the access point exposes as the root directory to NFS clients using the access point.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "tags_src",
+				Description: "The tags associated with the access point, presented as an array of Tag objects.",
+				Type:        proto.ColumnType_JSON,
+				Transform:   transform.FromField("Tags"),
+			},
+
+			// Standard columns for all tables
+			{
+				Name:        "tags",
+				Description: resourceInterfaceDescription("tags"),
+				Type:        proto.ColumnType_JSON,
+				Transform:   transform.From(getEfsAccessPointTurbotTags),
+			},
+			{
+				Name:        "title",
+				Description: resourceInterfaceDescription("title"),
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("Name"),
+			},
+			{
+				Name:        "akas",
+				Description: resourceInterfaceDescription("akas"),
+				Type:        proto.ColumnType_JSON,
+				Transform:   transform.FromField("AccessPointArn").Transform(arnToAkas),
+			},
+		}),
+	}
+}
+
+//// LIST FUNCTION
+
+func listEfsAccessPoints(ctx context.Context, d *plugin.QueryData, _ *plugin.HydrateData) (interface{}, error) {
+
+	// TODO put me in helper function
+	var region string
+	matrixRegion := plugin.GetMatrixItem(ctx)[matrixKeyRegion]
+	if matrixRegion != nil {
+		region = matrixRegion.(string)
+	}
+
+	// Create Session
+	svc, err := EfsService(ctx, d, region)
+	if err != nil {
+		return nil, err
+	}
+
+	// List call
+	err = svc.DescribeAccessPointsPages(
+		&efs.DescribeAccessPointsInput{},
+		func(page *efs.DescribeAccessPointsOutput, isLast bool) bool {
+			for _, accessPoint := range page.AccessPoints {
+				d.StreamListItem(ctx, accessPoint)
+			}
+			return !isLast
+		},
+	)
+
+	return nil, err
+}
+
+//// HYDRATE FUNCTIONS
+
+func getEfsAccessPoint(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
+
+	// TODO put me in helper function
+	var region string
+	matrixRegion := plugin.GetMatrixItem(ctx)[matrixKeyRegion]
+	if matrixRegion != nil {
+		region = matrixRegion.(string)
+	}
+
+	var accessPointID string
+	if h.Item != nil {
+		accessPointID = *h.Item.(*efs.AccessPointDescription).AccessPointId
+	} else {
+		quals := d.KeyColumnQuals
+		accessPointID = quals["access_point_id"].GetStringValue()
+	}
+
+	// create service
+	svc, err := EfsService(ctx, d, region)
+	if err != nil {
+		return nil, err
+	}
+
+	params := &efs.DescribeAccessPointsInput{
+		AccessPointId: aws.String(accessPointID),
+	}
+
+	data, err := svc.DescribeAccessPoints(params)
+	if err != nil {
+		return nil, err
+	}
+
+	if data.AccessPoints != nil && len(data.AccessPoints) > 0 {
+		return data.AccessPoints[0], nil
+	}
+	return nil, nil
+}
+
+//// TRANSFORM FUNCTIONS
+
+func getEfsAccessPointTurbotTags(ctx context.Context, d *transform.TransformData) (interface{}, error) {
```

Comment:
Don't use `get` prefix for the transform functions
```suggestion
func efsAccessPointTurbotTags(ctx context.Context, d *transform.TransformData) (interface{}, error) {
```